app/client/ui/errorPages.ts

```diff
@@ -98,6 +103,18 @@ export function createNotFoundPage(appModel: AppModel, message?: string) {
   ]);
 }
 
+export function createSigninFailedPage(appModel: AppModel, message?: string) {
+  document.title = t("Signin failed{{suffix}}", {suffix: getPageTitleSuffix(getGristConfig())});
```

Comment:
Oh, one more: We aren't everywhere consistent with this, but let's use a space for "sign in" and "log in" when used as a verb, and a hyphen for "sign-in" when used as a noun. So "Sign-in failed" here and in the next line, and then "Failed to log in".